### ISSUE TITLE
Trying to fix coverage 

### ISSUE THREAD

**Issue by yfxia** [berkeley-stat159/project-delta#118]
Hi, @jarrodmillman @matthew-brett @rossbar 

We are having some questions on make coverage. It turns out that we have written test files for diagnostics.py and convolution.py, but whenever we build travis CI, it skips the whole file so we basically are not getting any coverage from the above two files. Could you possibly help us figure out where goes wrong in those two test files? Thank you very much in advance.

**Comment by rossbar** [comment]
I took a look. If you run `make coverage` it shows that convolution.py is 23% covered and diagnostics.py is 0% covered. Taking a closer look at your test files, I noticed that test_convolution.py and test_diagnostics.py are the only two that don't have .pyc files, indicating they aren't being run. Those two files also have different permissions than all of the other files in the test directory. Changing the permissions seems to have helped some: try

```bash
chmod 664 test_convolution.py test_diagnostics.py
```

It may not completely solve your problem, but should take you in the right direction. Let me know if this helps and if you get re-stuck
